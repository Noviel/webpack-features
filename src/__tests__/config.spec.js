import configure from '../config';
import { envs } from '../fixtures';

let config = undefined;

describe('config', () => {
  it('should return list of all features', () => {
    config = configure(envs.modernBrowsersProd);
    expect(config).toMatchSnapshot();
  });
});
