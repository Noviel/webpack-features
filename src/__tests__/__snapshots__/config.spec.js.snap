// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`output should return correct object 1`] = `
Object {
  "output": Object {
    "chunkFilename": "[name].[chunkhash].chunk.js",
    "crossOriginLoading": false,
    "filename": "[name].[chunkhash].js",
    "path": "c:\\\\projects\\\\dev\\\\webpack-features\\\\dist",
    "publicPath": "/",
  },
}
`;
